front/src/modules/conflict/components/ConflictsListV2.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+
+import cx from 'classnames';
+import { useTranslation } from 'react-i18next';
+import { BsLightningFill } from 'react-icons/bs';
+
+import type { ConflictV2 } from 'common/api/osrdEditoastApi';
+import type { TrainScheduleWithDetails } from 'modules/trainschedule/components/TimetableV2/types';
+
+type ConflictTableProps = {
+  conflicts: ConflictV2[];
+  expanded: boolean;
+  trainSchedulesDetails: TrainScheduleWithDetails[];
+  toggleConflictsList: () => void;
+  onClick: (conflict: ConflictWithTrainNames) => void;
+};
+
+type ConflictWithTrainNames = ConflictV2 & {
+  train_names: string[];
+};
```

Comment:
Wouldn't it be better to put the additional type definitions in `common/api/osrdEditoastApi` ?

Reply:
Unfortunately,osrdEditoast is generated by the back, so we have to rely on the types defined.